init.lua

```diff
@@ -146,4 +146,24 @@ end
 -- for garbage collection
 cutorch.setHeapTracking(true)
 
+
+
+function torch.multinomialAliasSetup(probs, state)
+    local prob_tbl, alias_tbl
```

Comment:
simply this code as well, from comments in https://github.com/torch/torch7/pull/1046/files